lib/internal/cluster/master.js

```diff
@@ -232,7 +230,6 @@ cluster.disconnect = function(cb) {
     process.nextTick(() => intercom.emit('disconnect'));
   } else {
     for (var key in workers) {
```

Comment:
I think this should be changed to `for..of` as you have removed the below line which made this work. Though I'm not sure what's the current policy on such changes to `Module`.